main/ertp/ERTP-Introduction.md

```diff
@@ -0,0 +1,117 @@
+# ETRP Introduction
+
+ERTP (*Electronic Rights Transfer Protocol*) is Agoric's token
+standard for transferring tokens and other digital assets in
+JavaScript. Using the ETRP API, you can easily create digital assets,
+all of which are transferred exactly the same way and with exactly the
+same security properties. 
+
+ERTP uses object capabilities to enforce access control. If your
+program has a reference to an object, it can call methods on that
+object. If it doesn't have a reference, it can't. 
+
+## Creating assets with ERTP
+
+In ERTP, *mints* create digital *assets*. Access to an asset type's
+mint lets you create more digital assets of that type. You can then
+store new assets in a *payment* or a *purse*. 
+- **Payments**: Assets you intend to trade with someone else.
+- **Purses**: Store assets until you use them for a trade.
+
+To send assets in ERTP:
+1. Withdraw them from a purse.
+2. Put them in a payment.
+3. Send them to a recipient object as a message.
+
+To receive assets in ERTP:
+1. Create a purse for the asset type you'll receive.
+2. Get access to the asset type you'll receive. (**Note:** you
```

Comment:
I think 2 here is not right. The only way you create a purse is by having access to the asset type's issuer. So I think the steps should be:

```suggestion
1. Get access to the issuer for the asset type you expect to receive
2. Create a purse for that asset type
3. Receive the message with the payment and deposit the payment in your purse.
```